src/conf_mode/protocols_static.py

```diff
@@ -98,6 +98,15 @@ def generate(static):
     return None
 
 def apply(static):
+    ## Put routing table names in /etc/iproute2/rt_tables
+    with open("/etc/iproute2/rt_tables.d/vyos.conf", 'w') as f:
```

Comment:
Are there cases when we should delete this file?
What will be if we commit without saving and reboot the system?

Reply:
That's a good point. I also wonder if we should move clean-up to a separate script and move it to vyos-1x, or make the clean-up file list declarative and load it from a script — but that's a separate issue, of course.